src/main/java/com/labsynch/labseer/chemclasses/bbchem/ChemStructureServiceBBChemImpl.java

```diff
@@ -609,22 +621,48 @@ public StrippedSaltDTO stripSalts(CmpdRegMolecule inputStructure) throws CmpdReg
 	public HashMap<String, CmpdRegMolecule> standardizeStructures(HashMap<String, String> structures) throws CmpdRegMolFormatException {
 
 		// Get preprocessed structures (standardized mol structures)
-		HashMap<String, String> preprocessedStructures = new HashMap<String, String>();
+		HashMap<String, Entry<String, String>>preprocessedStructures = new HashMap<String, Entry<String, String>>();
 		try {
 			preprocessedStructures = bbChemStructureService.getPreprocessedStructures(structures);
 		} catch (IOException e) {
 			logger.error("Error getting preprocessed structures: "+e.getMessage());
 			throw new CmpdRegMolFormatException("Error getting preprocessed structures: "+e.getMessage());
 		}
 
+		// Only structures which have passed preprocessing should be passed to be processed so we need to remove those with an error
```

Comment:
This comment is a little misleading. We are not "removing" those with an error. We are substituting their original structure in lieu of the preprocessed structure (because we are missing a preprocessed structure). Could call this "filling in preprocessor failures with their original structure"